readme.md

```diff
@@ -1,7 +1,7 @@
 # Stars block
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://<cherniavskaaa/>.github.io/layout_stars/)
+- [TEST REPORT LINK](https://<cherniavskaaa/>.github.io/layout_stars/report/html_report/)
```

Comment:
Something is wrong with your links - please check them before submitting the task

Reply:
thank you, I fixed it)